ext/opcache/tests/file_cache_error.phpt

```diff
@@ -0,0 +1,42 @@
+--TEST--
+File cache error 001
+--EXTENSIONS--
+opcache
+posix
+pcntl
+--INI--
+opcache.enable_cli=1
+opcache.file_cache={TMP}
+opcache.log_verbosity_level=2
+--SKIPIF--
+<?php
+posix_setrlimit(POSIX_RLIMIT_FSIZE, 1, 1) || die('skip Test requires setrlimit(RLIMIT_FSIZE) to work');
+ini_parse_quantity(ini_get('opcache.jit_buffer_size')) === 0 || die('skip File cache is disabled when JIT is on');
```

Comment:
I, personally, dislike this pattern (regardless of whether `||` or `or` is used), and would go with a single line `if` statement.

Reply:
Same. Although the style guide says you should add braces to `if` statements.

Reply:
Then I'd rather change the style guide, than to find some loopholes. :)

Reply:
I agree, I don't find single line `if`s to be an issue. IMO the only justified concern is: 

```c
#define FOO() foo(); bar()

if (baz) FOO();
``` 

Most of our macros guard against this with a `do {} while(0);` loop though.